src/index.js

```diff
@@ -1 +1,13 @@
 import './style.css';
+import Players from './functions.js';
+
+Players.createNewGame();
```

Comment:
### OPTIONAL 

I assume creating a new game every instance was on purpose and it does work really well but it would be nice to have persistent data on the API 🤓 What do you think if the game is created once and that way we maintain the data to have a cooler app 😎?